### DISCUSSION TITLE
2023년 06월 28일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#62] [General]
# 2023년 06월 28일

[ 2023-06-01 ] [ 2023-06-03 ] [ 2023-06-06 ] [ 2023-06-07 ] [ 2023-06-26 ] [ 2023-06-28 ]

https://vero.wiki/2023-06-28/

**Comment by kevstevie** [comment]
아니 상현이형 팀 왜 혼자 들어감?

**Reply by Cyma-s** [reply]
아니 선릉 왜 혼자 감?;;

**Comment by drunkenhw** [comment]
내 슉하고 싶은데 교환하실?

**Reply by Cyma-s** [reply]
어림도 없지 ㅋㅋ

**Comment by hong-sile** [comment]
방학 끝나고 살아난 베로 위키 ㄷㄷ

**Reply by Cyma-s** [reply]
베로 위키 앞으로도 많관부
